src/com/company/Main.java

```diff
@@ -1,46 +1,82 @@
 package com.company;
 import java.io.*;
-import java.util.ArrayList;
-import java.util.List;
+import java.nio.file.Files;
 import java.util.Scanner;
+//import static org.junit.Assert.assertTrue;
 
 
 public class Main {
 
-    static File fileToRead;
-    static File fileToSave;
-    static FileWriter fileWriter;
-    static List<Integer> unsortedNumbers = new ArrayList<>();
-    public static List<Integer> numbersToSave;
+    private static File fileToRead;
+    private static File fileToSave;
+    public static Scanner scanner;
 
-    public static void main(String[] args) throws IOException {
+    public static void main(String[] args) {
 
-        System.out.println("Path to unsorted numbers file:\n");
+        scanner = new Scanner(System.in);
+        String fileNameToRead = getFirstParam("Specify path to file with unsorted numbers", scanner);
+        String directoryNameToSaveSortedData = getDirectoryToSave("Specify path to directory to save sorted data", scanner);
+        String fileNameToSaveSortedData = getFileNameToSave("Specify file name to save sorted data", scanner);
```

Comment:
You have 2 input args, not 3. fileNameToRead and fileNameToSaveSortedData

Reply:
done

Reply:
Fixed. There are just two args:  fileNameToRead and fileNameToSaveSortedData